spec/pane-axis-element-spec.coffee

```diff
@@ -0,0 +1,58 @@
+PaneAxisElement = require '../src/pane-axis-element'
+
+describe "PaneResizeHandleElement", ->
+  describe "add and remove", ->
+    class TestItemElement extends HTMLElement
+      createdCallback: ->
+        @classList.add('test-root')
+
+    TestItemElement = document.registerElement 'atom-test-item-element', prototype: TestItemElement.prototype
```

Comment:
I think you should be able to use a generic `div` element to test this rather than registering your own. Adding a custom element here adds complexity and it also can never be registered with the current custom elements API.